### ISSUE TITLE
Create a rendered version of the sci website

### ISSUE THREAD

**Issue by jawache** [Green-Software-Foundation/sci#368]
@osamajandali can you explore creating a version of this website on sci.greensoftware.foundation.

- It should take the main branch and render just the main spec file, it's just one page.
- The headings should be linkable.
- It should have a menu to link to the github repo.
- it should have a menu to link to the standards working group wiki page and the sci wiki page.
- it should have a menu item to link to the green software foundation.
- This task shouldn't add a lot of code to **this** repo, this repo should be kept as clean as possible to focus on the written content of the website.

Additional task:
- [ ] Add a link to this website from the main GSF site, `Resources` menu

**Comment by osamajandali** [comment]
- Can you tell me exactly which "main spec file" are you referring to ?
- Do I need to render only one markdown file or all the markdown files into one page?

If it is only one page I can do a script that convert a markdown into a html.

Otherwise, if it will contain routing and navigating throught multiple pages and you want to you autogenerate pages from markdown, I use a framework called astro that is very straight forward and doesn't add a lot of code not configuration to get started. I used in [Speakers](https://github.com/Green-Software-Foundation/speakers) website

**Comment by jawache** [comment]
This file in particular (https://github.com/Green-Software-Foundation/sci/blob/dev/Software_Carbon_Intensity/Software_Carbon_Intensity_Specification.md) the actual location of this file might change (might move to the root).

It will be just one file + header + footer that we add (with menu/links to other places). No sub-pages etc...

**Comment by zanete** [comment]
Capturing status of weekly call:
* Still working on this, should be finished by this week.

**Comment by osamajandali** [comment]
Done 
@jawache 
- See the render version here: https://green-software-foundation.github.io/sci/
- I kept everything so simple, no framework in the code.
- It converts SPEC.md file when changes into html using github actions.
- Check the code on `webpage` branch and if everything is okay, I'll merge it into dev
- Shall I create and use sci.greensoftware.foundation custom domain ?

**Comment by zanete** [comment]
@osamajandali - please also add GA tracking to this when deploying 🙏

**Comment by jawache** [comment]
Thanks @osamajandali 
- yes please make a sci.gsf domain.
- create a PR and leave it for the WG to merge once they review in their weekly meetings.
- Needs the version number somewhere, not sure where you are going to get that from, the branch name? @seanmcilroy29 where is the version stored?
- Apologies but realized I think we need some text at the top that helps people understand what this is. In fact I think we can lose the menu and just have copy like the below.

Version XXXX of the [Software Carbon Intensity (SCI)](https://greensoftwarefoundation.atlassian.net/wiki/spaces/~612dd45e45cd76006a84071a/pages/15958033/Standards+SCI) specification by the [Standards Working Group](https://greensoftwarefoundation.atlassian.net/wiki/spaces/~612dd45e45cd76006a84071a/pages/819277/Standards+Working+Group) in the [Green Software Foundation](https://greensoftware.foundation/). The SCI technical specification, describes how to calculate the carbon intensity of a software application. It describes the methodology of calculating the total carbon emissions and the selection criteria to turn the total into a rate that can be used to achieve real-world, physical emissions reductions. To see the latest version, raise an issue or ask a question visit the [GitHub](https://github.com/Green-Software-Foundation/sci) repository .

**Comment by seanmcilroy29** [comment]
@jawache / @osamajandali - The Spec version release will be tagged v1.0 in the repo once ISO accepts the submitted minor editorial updates and assigns their Standard ISO number. This'll ensure version synchronization between GSF and ISO.

**Comment by jawache** [comment]
@seanmcilroy29 can we have some front-matter in the spec.md, so basically add this to the top of the file.

---
version: 1.0.0
---

That isn't rendered but allows @osamajandali to read the version number and put it on the website.

I've created a PR to add this, the WG will need to approve (hopefully this is allowed for ISO?) https://github.com/Green-Software-Foundation/sci/pull/372

**Comment by zanete** [comment]
@jawache @seanmcilroy29 can you confirm we're waiting for the WG to approve the PR before Osama can take next action?

**Comment by zanete** [comment]
@seanmcilroy29 cc @jawache are there any news on this?

**Comment by seanmcilroy29** [comment]
@zanete - I'd prefer to tag the current main branch as v1.0 until after the Standard has been given its ISO number. It's unlikely, but If we tagged the main branch as v1.0 now and ISO came back asking for some additional editorials, then our semantic versioning would be out of Sync with the ISO releases.

My hope is that ISO offer some updates in Nov regarding this

**Comment by jawache** [comment]
Hey @seanmcilroy29 FYI this isn't about tagging the main branch with a version number it's about adding some meta data to the spec so Osama can print out the version number in the website. Even if you tagged the branch he can't read that to print the version number, it needs some front matter in the spec file.

This is the only thing blocking us from having a website for the sci. This isn't about tagging the branch, you can always edit the version number later to match the version that ISO gives us.

Just need this merged https://github.com/Green-Software-Foundation/sci/pull/372 that will unblock Osama. When ISO comes back with a version number we just update it to whatever ISO provides us.

**Comment by zanete** [comment]
@seanmcilroy29 When you get back, could you let us know what the next steps on this could be so we can plan when we can action?

**Comment by zanete** [comment]
@seanmcilroy29 a ping to get your attention to the commend above ☝

**Comment by seanmcilroy29** [comment]
@zanete - https://github.com/Green-Software-Foundation/sci/pull/372 has been merged into dev. Osama should be unblocked to complete the webpage

**Comment by zanete** [comment]
Capturing status in the web call: 
- will continue this week

**Comment by osamajandali** [comment]
Done!
https://sci.greensoftware.foundation/
It is also merged in the `dev` branch so that you can edit directly from there now

**Comment by zanete** [comment]
thanks @osamajandali ! @jawache can you review and confirm it's all as per your request? 🙏

**Comment by jawache** [comment]
Thank you this looks perfect!

 @NAMRATA-WOKE FYI we now have a better link to send people if they want to read the SCI spec.

**Comment by seanmcilroy29** [comment]
@osamajandali - To make this easily accessible, can we add a link to 'Resources' drop down menu?

**Comment by jawache** [comment]
@seanmcilroy29 you mean the resources menu on the main GSF website?

**Comment by seanmcilroy29** [comment]
@jawache - yes, should have been more specific

**Comment by jawache** [comment]
😜 yeah good idea!

**Comment by osamajandali** [comment]
Added
